backend/src/services.ts

```diff
@@ -0,0 +1,23 @@
+import Logger from './Logger'
+import EventBus from './EventBus'
+const isPortReachable = require('is-port-reachable')
+
+const EVENTS = {
+  serviceCheckResponse: 'service/check-response',
+}
+
+export const checkHostAndPort = async (data: any) => {
+  Logger.info('PING', { data })
+  let isValid = false
+  try {
+    const { port, host } = data
+    isValid = await isPortReachable(port, { host })
+    Logger.info('IS PORT REACHABLE?', { isValid })
+  } catch (error) {
+    Logger.info('IS NOT VALID', { error })
+    isValid = false
+  }
+  EventBus.emit(EVENTS.serviceCheckResponse, isValid)
+}
+
+export default { EVENTS }
```

Comment:
We already have a portscanner file. If they are different, then maybe this 'service' page should be added to the portscanner page. Also let's not add the websocket event to the 'service/' namespace - those are all for connection communication. We can simply use a single new namespace 'reachablePort'